decidim-comments/app/models/decidim/comments/comment.rb

```diff
@@ -26,21 +25,23 @@ class Comment < ApplicationRecord
       MAX_DEPTH = 3
 
       translatable_fields :body
+
       belongs_to :commentable, foreign_key: "decidim_commentable_id", foreign_type: "decidim_commentable_type", polymorphic: true
-      belongs_to :root_commentable, foreign_key: "decidim_root_commentable_id", foreign_type: "decidim_root_commentable_type", polymorphic: true
+      belongs_to :root_commentable, foreign_key: "decidim_root_commentable_id", foreign_type: "decidim_root_commentable_type", polymorphic: true, touch: true
       has_many :up_votes, -> { where(weight: 1) }, foreign_key: "decidim_comment_id", class_name: "CommentVote", dependent: :destroy
       has_many :down_votes, -> { where(weight: -1) }, foreign_key: "decidim_comment_id", class_name: "CommentVote", dependent: :destroy
 
+      after_save :update_counter
+      after_destroy :update_counter
+      after_touch :update_counter
```

Comment:
I was going to say to update the root commentable, but it doesn't make any sense with the current code. Maybe I forgot to remove it!

Reply:
Now I've remembered! When a resource is reported and its moderation is hidden, the reported content is only touched, but not updated. So when a comment is hidden because the moderation is updated we need to notify the root commentable about this.

Reply:
I've added docs